synapse/cryotank.py

```diff
@@ -47,68 +47,32 @@ async def __anit__(self, dirn, conf=None):
         self.conf = conf
         self.dirn = s_common.gendir(dirn)
 
-        self._iden = self._getTankIden()
+        self._iden = iden
 
         path = s_common.gendir(self.dirn, 'tank.lmdb')
 
         self.slab = await s_lmdbslab.Slab.anit(path, map_async=True, **conf)
 
-        self.offs = s_slaboffs.SlabOffs(self.slab, 'offsets')
         self._items = s_slabseqn.SlabSeqn(self.slab, 'items')
         self._metrics = s_slabseqn.SlabSeqn(self.slab, 'metrics')
 
         self.onfini(self.slab.fini)
 
-    async def iden(self):
+    def iden(self):
```

Comment:
Does this need to be a method? Why not just a regular python property?

Reply:
I am guessing the iden accessor was added since there is a tank telepath api iden().  So no, not explicitly needed, but I also didn't see any reason to remove it.

Reply:
For reference on Telepath APIs objects, we cannot use `@property` getters / setters. 

While we could do a `tank.iden` reference for the heavy Crytotank object, we do start having a schism between the CryoTank and CryoAPI for accessing the iden.